lib/formtools/render-overview.js

```diff
@@ -0,0 +1,88 @@
+
+var async = require('async'),
+    cellRenderers = require('./renderers-cell');
+
+
+var mapCellRenderer = function (type) {
+
+    //add text, password, enum and hidden function in renderers-cell.js
+
+    var cellrenderer;
+
+    switch (type) {
+
+        case 'email':
+            cellrenderer = 'text';
+            break;
+
+        case 'number':
+            cellrenderer = 'text';
+            break;
+
+        case 'digit':
+            cellrenderer = 'text';
+            break;
+
+        case 'tel':
+            cellrenderer = 'text';
+            break;
+
+        case 'documentarray':
+            cellrenderer = 'documentArray';
+            break;
+
+        case 'datetime':
+            cellrenderer = 'dateTime';
+            break;
+
+        case 'datetimeLocal':
+            cellrenderer = 'localDateTime';
+            break;
+
+        default:
+            cellrenderer = 'default';
+            break;
+
+    }
+
+    return cellrenderer;
+};
+
+var generateViewFromModel = exports.generateViewFromModel = function (schema, schemaFields, record, model, callback) {
```

Comment:
A better name for this function is perhaps `render`? And you should structure it like the example below:

```
function render () {};
module.exports = {
  render: render
}
```